### PR TITLE
Show hint to Chinese users

### PR DESCRIPTION
**Pull request by timqian**
For users in China, show a more obvious hint about the `sls --help-comonents`

Do you have any suggestions about the wording? @ole3021 

## Preview
Screen Shot 2021-03-19 at 10 52 19 AM

### CODE REVIEW

File: lib/classes/CLI.js [hunk 1]
```diff
@@ -128,6 +129,13 @@ class CLI {
   generateMainHelp() {
     this.consoleLog('');
 
+    if (isInChina) {
```
Comment (medikoo) [inline lib/classes/CLI.js:132]: @timqian Below we already have section in which we document `--help-components` I believe we should not show two of them. 

I think it'll be best if we hide below one at same time

**Review by medikoo — CHANGES_REQUESTED** [review]
Good call @timqian I just propose to not double `--help-components` hint

File: lib/classes/CLI.js (hunk 1 above)
Comment (timqian) [reply lib/classes/CLI.js:132]: Just removed the duplicated info

**Comment by pgrzesik** [comment]
Hello @timqian - is this PR still valid?

File: lib/classes/CLI.js [hunk 2]
```diff
@@ -128,6 +129,13 @@ class CLI {
...
+      this.consoleLog(
+        '检测到您在中国，如果您是 Serverless Components 用户，请使用 "serverless --help-components" 命令查看相关命令'
```
Comment (ole3021) [inline lib/classes/CLI.js:134]: 如果您是 Tecnent Serverless 用户，请使用 "serverless --help-components" 命令查看相关命令

**Review by ole3021 — COMMENTED** [review]
@pgrzesik  thanks for asking, we are good to close this PR. as 

* Currently we have add more commands by default will be handled by Components for users from China. 
* in future we will build a plugin like serverless-azure to fully overwrite the CLI for user from China.

**Comment by pgrzesik** [comment]
Thanks @ole3021 :bow:

**Pull request closed without merging**